gtk/README.md

```diff
@@ -83,6 +83,16 @@ When opening a PR please put the changes to the `src/auto` directory in a separa
 You may also wish to run `cargo clippy -- -D warnings` and check that you're clean because
 otherwise you may be surprised when CI fails.
 
+## See Also
+
+ * [atk](https://crates.io/crates/atk)
+ * [cairo](https://crates.io/crates/cairo-rs)
+ * [gdk](https://crates.io/crates/gdk)
+ * [gdk-pixbuf](https://crates.io/crates/gdk-pixbuf)
```

Comment:
could mention gdk-x11 as well, as none of the "See also" can lead you to it

Reply:
I only listed dependencies every time.